src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -70,9 +70,35 @@ public ReplicatorTest()
             ReopenDB();
             Database.Delete("otherdb", Directory);
             _otherDB = OpenDB("otherdb");
+            Database.SetLogLevel(LogDomain.Replicator, LogLevel.Verbose);
```

Comment:
This was meant to be temporary.  If left in, all the logs for the rest of the test suite will be at this level